org-policy/user_permissions.rego

```diff
@@ -0,0 +1,26 @@
+package diamond
+
+import rego.v1
+
+profile := http.send({
+	"url": opa.runtime().env.PROFILE_ENDPOINT,
+	"method": "GET",
+	"headers": {"authorization": input.attributes.request.http.headers.authorization},
+})
+
+subject := profile.sub
+
+# METADATA
+# entrypoint: true
+user_on_session(proposalNumber, visitNumber) if {
+	some allowed_session in data.diamond.data.users.sessions[subject]
+	allowed_session[0] = proposalNumber
+	allowed_session[1] = visitNumber
```

Comment:
Could we simply perform a set check here? Also, looks like the [unification](https://www.openpolicyagent.org/docs/latest/policy-language/#unification-) operator is generally discouraged

```suggestion
	[proposalNumber, visitNumber] in data.diamond.data.users.sessions[subject]
```